editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/flag/JFlag.java

```diff
@@ -0,0 +1,74 @@
+package ca.ubc.eml.soiltopargraphy.editor.ui.flag;
+
+import ca.ubc.eml.soiltopargraphy.editor.ui.infopanel.JInfoPanel;
+
+import java.util.ArrayList;
+
+
+
+public class JFlag {
+
+    private String location;
+    private float lattitude;
+    private float longditude;
+    private ArrayList<JInfoPanel> infoPanels;
```

Comment:
There's 1 info panel associated with 1 flag, so instead of ArrayList, just have 1 InfoPanel.